### ISSUE TITLE
TextContext.Progress.Writeline does not work in 4.4.2 when running dotnet test

### ISSUE THREAD

**Issue by tchengwhisker** [nunit/nunit3-vs-adapter#1089]
NUnit 3.13.3
NUnit3TestAdapter 4.4.2
Visual Studio Enterprise 2022 Version 17.4.4
dotnet --version is 7.0.102

Running dotnet test from CLI to execute code:
TestContext.Progress.WriteLine("project setup");

Outputs "project setup" in console when running NUnit3TestAdapter 4.3.1. Changing to version 4.4.2 in the same project, rebuilding, and running the same CLI yields no output for any TestContext.Progress.WriteLine calls.

**Comment by OsirisTerje** [comment]
There have been several changes to this over time.  Need to get this properly documented and checked.  Thanks for reporting!

**Comment by jacobilsoe** [comment]
I am also seeing this issue. Are there any known workarounds?

**Comment by OsirisTerje** [comment]
This was changed in 4.4 by #1037 .  
In order to get the information out, you need to add the standard logger:
```cmd
dotnet test --logger "Console;Verbosity=normal"
```

TL;DR;
The reason is that the Progress messages come out as Informational information, and that now requires a logger. We did earlier output it as Warning information, but the new Test Explorer now gives warnings when that happens, so that "workaround" doesn't work anymore.
This needs to be better documented of course.
